src/test/datascience/kernelLauncher.vscode.test.ts

```diff
@@ -30,7 +30,7 @@ suite('DataScience - Kernel Launcher', () => {
         language: 'python',
         display_name: 'Python 3',
         metadata: {},
-        argv: ['python', '-m', 'ipykernel_launcher', '-f', `{connection_file}`],
+        argv: [PYTHON_PATH, '-m', 'ipykernel_launcher', '-f', `{connection_file}`],
```

Comment:
Found a weird issue, an error with ipykernel missing, i guess `python` was pointing to some other interpreter, either way this is a safe change (always use the designated interpreter)